### DISCUSSION TITLE
Gaussian variogram different from Webster, 2007

### DISCUSSION THREAD

**Discussion by rlsandovalp** [GeoStat-Framework/GSTools#306] [Q&A] [answered]
Hi,

Just wondering why the Gaussian variogram differs from the equation written by Webster (2007).

There, the equation reads 
$\gamma (h) = c\left(1-\exp\left(-\frac{h^2}{r^2}\right)\right)$

However, in the covmodels file it is written as

$\gamma(h)=       c       \left(1-\exp\left(-\left(s\cdot\frac{h}{r}\right)^{2}\right)\right)$

Where the standard rescale factor is $s=\frac{\sqrt{\pi}}{2}$.

**Comment by MuellerSeb** [comment, marked as answer]
There are different formulations of the gaussian covariance model that basically only differ by the rescale factor and this was also the reason why we added the scaling factor.

We choose this factor so the length scale actually matches the integral scale of the model.

If you want the formulation of webster, just set `s=1`.

**Comment by LSchueler** [comment]
[Here](https://geostat-framework.readthedocs.io/projects/gstools/en/latest/examples/02_cov_model/04_different_scales.html#sphx-glr-examples-02-cov-model-04-different-scales-py) you can see a few different scales, which are commonly used.
